### PR TITLE
Campaign finder  error messages

### LINKED ISSUE #1731: Campaign Finder - Network connectivity error state
For when the user isn't online, solr search is down, etc.

### PR DESCRIPTION
**Pull request by weerd**
Resolves #1731

@dave take a gander. I ended up placing the messages element code in the SolrAdaptor file, which is probs not ideal, but works for now. Maybe address during refactoring sprint, etc.

### CODE REVIEW

**Comment by weerd** [comment]
Chatted with @DFurnes and going to abstract the data a bit to better organize it.

stay tuned...

**Comment by weerd** [comment]
Ok! PR take two!

File: lib/themes/dosomething/paraneue_dosomething/js/finder/SolrAdapter.js [hunk 1]
```diff
@@ -65,15 +69,16 @@ define(["jquery", "lodash"], function($, _) {
      * Run the query after a specified timeout.
      *
      * @param {jQuery} $fields  Array of fields selected in the interface.
-     * @param successCallback   Function called when query successfully completes.
+     * @param responseCallback   Function called when query successfully completes.
      */
-    throttledQuery: function($fields, successCallback) {
+    throttledQuery: function($fields, responseCallback) {
+      debugger;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/SolrAdapter.js]: :bangbang:

File: lib/themes/dosomething/paraneue_dosomething/js/finder/SolrAdapter.js [hunk 2]
```diff
@@ -131,7 +136,16 @@ define(["jquery", "lodash"], function($, _) {
         jsonpCallback: 'solrResponse',
         url: SolrAdapter.buildQuery(query, q),
         success: function (data) {
-          successCallback(data);
+          responseCallback({
+            result: data,
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/SolrAdapter.js:139]: Should this be `result: data.result` to keep the same interface elsewhere?

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/finder/SolrAdapter.js:139]: It's still passing the data correctly. `data.result` gets used in the **Finder.js** file, and consequently gets passed along the same as before from that point on.

**Pull request merged**